test/promql/tests/test_progressing_argocd.yaml

```diff
@@ -0,0 +1,81 @@
+evaluation_interval: 1m
+
+rule_files:
+  - ../extracted-rules.yaml
+
+tests:
+  - interval: 1m
+    input_series:
+      # app is progessing and not healthy, should alert
+      - series: 'argocd_app_info{health_status="Progressing", name="first-test-app", dest_namespace="Still-Progressing"}'
+        values: 1x19
+      - series: 'argocd_app_info{health_status="Healthy", name="first-test-app", dest_namespace="Still-Progressing"}'
+        values: 0x19
+    alert_rule_test:
+      - eval_time: 20m
+        alertname: ProgressingArgocdApp
+        exp_alerts:
+          - exp_labels:
+              severity: Critical
+              dest_namespace: Still-Progressing
+              health_status: Progressing
+              name: first-test-app
+            exp_annotations:
+              summary: "Application first-test-app in namespace Still-Progressing is in 'Progressing' state for more than 20 minuts"
+              description: "Application first-test-app in namespace Still-Progressing is in 'Progressing' state for more than 20 minuts"
+
+  - interval: 1m
+    input_series:
+      # app was progessing and then Healthy for the last 5 min, should not alert
+      - series: 'argocd_app_info{health_status="Progressing", name="second-test-app", dest_namespace="Progressing-Healthy-5min"}'
+        values: 1x14 0x4
```

Comment:
As discussed, all the `0` should be updated to `_` (all over the tests)

Reply:
Done